src/palette/internal/palette.cpp

```diff
@@ -515,102 +508,143 @@ Palette::Type Palette::guessType() const
         }
     }
 
+    return guessTypeFrom(e);
+}
+
+Palette::Type guessTypeFrom(const EngravingItem* e)
+{
     if (!e) {
-        return Type::Custom;
+        return Palette::Type::Custom;
     }
 
     switch (e->type()) {
     case ElementType::CLEF:
-        return Type::Clef;
+        return Palette::Type::Clef;
     case ElementType::KEYSIG:
-        return Type::KeySig;
+        return Palette::Type::KeySig;
     case ElementType::TIMESIG:
-        return Type::TimeSig;
+        return Palette::Type::TimeSig;
     case ElementType::BRACKET:
     case ElementType::BRACKET_ITEM:
-        return Type::Bracket;
+        return Palette::Type::Bracket;
     case ElementType::ACCIDENTAL:
-        return Type::Accidental;
+        return Palette::Type::Accidental;
     case ElementType::ARTICULATION:
+        if (toArticulation(e)->isOrnament()) {
+            return Palette::Type::Ornament;
+        } else if (toArticulation(e)->isLuteFingering()) {
+            return Palette::Type::Fingering;
```

Comment:
Lute fingerings are implemented as articulations, but belong to the "fingerings" palette.